### PR TITLE
[refactor-prompt] Migrate command: wallet import token

### PR DESCRIPTION
**Pull request by LysanderGG**
**What current issue(s) does this address, or what feature is it adding?**
#623 
Migrate the command `wallet import token` to the new `Command` system.

**How did you solve this problem?**
Implementation of `CommandWalletImportToken`

**How did you make sure your solution works?**
unit tests & manual testing

**Are there any special changes in the code that we should be aware of?**
No

**Please check the following, if applicable:**

- [x] Did you add any tests?
- [x] Did you run `make lint`?
- [x] Did you run `make test`?
- [x] Are you making a PR to a feature branch or development rather than master?
- [ ] Did you add an entry to `CHANGELOG.rst`? (if not, please do)

### CODE REVIEW

File: neo/Prompt/Commands/Wallet.py [hunk 1]
```diff
@@ -620,6 +621,23 @@ def command_desc(self):
...
 
 
+class CommandWalletImportToken(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        if len(arguments) != 1:
+            print("Please specify the required parameter")
+            return None
```
Comment (jseagrave21) [inline neo/Prompt/Commands/Wallet.py]: Is there a reason to specify `return None`? Previously we have just used `return`. See https://github.com/CityOfZion/neo-python/blob/b54648a5552f8ded02b98ddccf95b1a23560f640/neo/Prompt/Commands/Wallet.py#L89-L95 etc.
Similary, I would remove `None` here https://github.com/CityOfZion/neo-python/blob/b54648a5552f8ded02b98ddccf95b1a23560f640/neo/Prompt/Commands/Wallet.py#L286

File: neo/Prompt/Commands/Wallet.py [hunk 2]
```diff
@@ -669,7 +687,7 @@ def DeleteAddress(wallet, addr):
 def ImportToken(wallet, contract_hash):
     if wallet is None:
         print("please open a wallet")
-        return False
+        return None
```
Comment (jseagrave21) [inline neo/Prompt/Commands/Wallet.py]: Similarly, I would remove `None`

File: neo/Prompt/Commands/Wallet.py (hunk 1 above)
Comment (LysanderGG) [reply neo/Prompt/Commands/Wallet.py]: Fixed, thank you 👍

**Comment by ixje** [comment]
Needs conflict resolving.

**Comment by LysanderGG** [comment]
> Needs conflict resolving.

@ixje Done!

File: neo/Prompt/Commands/Wallet.py [hunk 3]
```diff
@@ -738,7 +756,7 @@ def DeleteAddress(wallet, addr):
 def ImportToken(wallet, contract_hash):
     if wallet is None:
         print("please open a wallet")
-        return False
+        return
 
     contract = Blockchain.Default().GetContract(contract_hash)
```
[the remark this answers is not in the document]
Comment (LysanderGG) [reply neo/Prompt/Commands/Wallet.py]: Thank you, fixed with 8e1b6cf

**Comment by ixje** [comment]
great! with this just 1 more group to go I think 💯

**Pull request merged**